src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Find your way with field data in the Web Vitals extension
```

Comment:
I'd suggest a more straightforward title, rather than a clever one. You want people to know exactly what's the post about before clicking it. How about something like "Field data comes to the Web Vitals extension"?

Reply:
I actually like the clever title. It alludes to the value of field data and it helps tie together a metaphor used in the post with hero image.

Reply:
Here's my reasoning: consider all the different ways people can reach a link to this post without the context of the hero image such as newsletters, hacker news style aggregators, etc. The image can also fail to load, and then there are people with poor vision. The title is cute and clever, but it obfuscates what the post is about. I wasn't sure until the second paragraph. Not saying you can't have fun, but in this case alternatives would be more effective.

If you still want to keep it, I think "Find your way with field data in the Web Vitals extension update" is better than "new field data".